packages/relay/tests/lib/utils.spec.ts

```diff
@@ -190,17 +192,57 @@ describe('Utils', () => {
       '0x127': 'mainnet',
       '0x128': 'testnet',
       '0x129': 'previewnet',
-      '0x12a': 'local'
+      '0x12a': 'local',
     })) {
-      withOverriddenEnvsInMochaTest({
-          CHAIN_ID: chainId
-        }, () => {
+      withOverriddenEnvsInMochaTest(
+        {
+          CHAIN_ID: chainId,
+        },
+        () => {
           it(`should return ${networkName} for chain id ${chainId}`, () => {
             const networkName = Utils.getNetworkNameByChainId();
             expect(networkName).to.equal(networkName);
           });
-        }
+        },
       );
     }
   });
+
+  describe('arrangeRpcParams', () => {
```

Comment:
Should we also add tests for `traceTransaction` here since it's the most complex case?